persistence/mongodb-record/src/main/scala/net/liftweb/mongodb/record/MongoRecordRules.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 WorldWide Conferencing, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.liftweb
+package mongodb
+package record
+
+import net.liftweb.mongodb.record.codecs.RecordCodec
+import net.liftweb.util.SimpleInjector
+
+import org.bson.Transformer
+import org.bson.codecs.BsonTypeClassMap
+import org.bson.codecs.configuration.CodecRegistry
+
+object MongoRecordRules extends SimpleInjector {
```

Comment:
Some Scaladoc here would be helpful for folks just trying to figure out what this is at the class level.

Reply:
Good idea. I added a short description.